linter.py

```diff
@@ -11,25 +11,20 @@
 """This module exports the Lintr plugin class."""
 
 from SublimeLinter.lint import Linter, util
-import os
 
 
 class Lintr(Linter):
-
```

Comment:
I got back the line, but still Travis check fails. The error message was:

```
The command "flake8 . --max-line-length=120" exited with 0.
$ pep257 . --ignore=D202
./linter.py:16 in public class `Lintr`:
        D211: No blank lines allowed before class docstring (found 1)
```

I don't know about flake8 deeply, but it looks that D203 and D211 are conflicting and we have to ignore either. Which should I follow?